docs/next/modules/en/pages/tutorials/uninstall.adoc

```diff
@@ -28,6 +15,15 @@ This may take a few minutes to complete.
 Remember that, if you use a different name for the installation or a different namespace, you may need to customize the command for your specific configuration.
 ====
 
+[CAUTION]
+====
+When installing {product_name} in your Rancher environment, by default, {product_name} enables the Cluster API Operator cleanup. This includes cleaning up Cluster API Operator specific webhooks and deployments that otherwise cause issues with Rancher provisioning.
+
+To simplify uninstalling {product_name} (via Rancher Manager or helm command), the official {product_name} Helm chart includes a `post-delete` hook that applies these changes, making it transparent to the end user:
+
+* Delete the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks that are no longer needed.
+* Delete the CAPI `deployments` that are no longer needed.
```

Comment:
```suggestion
To simplify uninstalling {product_name} (via Rancher Manager or helm command), the official {product_name} Helm chart includes a `post-delete` hook that applies the following changes, making it transparent to the end user:

* It deletes the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks that are no longer needed.
* It deletes the CAPI `deployments` that are no longer needed.
```

I suggested this change (`Delete` to `It deletes`) to make it clear there's no ask for the user to act on this.